tests/mssqltestutils.py

```diff
@@ -137,3 +186,21 @@ def get_io_paths(test_file_suffix):
     i = os.path.join(_BASELINE_DIR, 'test_query_inputs', 'input_%s' % test_file_suffix)
     o = os.path.join(_BASELINE_DIR, 'test_query_baseline', 'baseline_%s' % test_file_suffix)
     return (i, o)
+
+
+class TestDB:
+    """
+    TestDB creates a test db for any tests inheriting this class, and cleans up resources when
+    the module is complete.
+    """
+    @staticmethod
+    @pytest.fixture(scope='module')
+    def test_db():
```

Comment:
creating an iterator for a single element to injecting the client seems overkill. Can we just have a setup method with creates the client?

Reply:
I'm confused, I don't believe this is an iterator.

We're using Pytest [fixtures](https://docs.pytest.org/en/latest/fixture.html#scope-sharing-a-fixture-instance-across-tests-in-a-class-module-or-session) for tests which were already using this behavior, I was just trying to reduce redundant code.